tests/tests_unit/rules/test_models/test_asset_rules.py

```diff
@@ -0,0 +1,362 @@
+from datetime import datetime
+from typing import Any
+
+import pandas as pd
+import pytest
+from cognite.client import data_modeling as dm
+
+from cognite.neat.rules.models import DMSRules, SheetList
+from cognite.neat.rules.models._constants import DMS_CONTAINER_SIZE_LIMIT
+from cognite.neat.rules.models.asset import (
+    AssetClass,
+    AssetRules,
+    AssetRulesInput,
+)
+from cognite.neat.rules.models.asset._converter import _AssetRulesConverter
+from cognite.neat.rules.models.asset._rules_input import (
+    AssetClassInput,
+    AssetMetadataInput,
+    AssetPropertyInput,
+)
+from cognite.neat.rules.models.data_types import DataType
+from cognite.neat.rules.models.entities import AssetEntity, RelationshipEntity
+from cognite.neat.utils.spreadsheet import read_individual_sheet
+from tests.config import DOC_RULES
+
+
+@pytest.fixture(scope="session")
+def david_spreadsheet() -> dict[str, dict[str, Any]]:
+    filepath = DOC_RULES / "information-architect-david.xlsx"
+    excel_file = pd.ExcelFile(filepath)
```

Comment:
We should probably move this to a central place in our test. Reading excel files can be slow and slows down the overall running of our test suite.